corehq/apps/style/static/style/js/pagination.ng.js

```diff
@@ -0,0 +1,121 @@
+(function (angular, undefined) {
+    'use strict';
+
+    var pagination = angular.module('hq.pagination', [
+        'ui.bootstrap',
+        'ngResource',
+        'ngRoute',
+        'ngCookies',
+        'ng.django.rmi'
+    ]);
+
+    pagination.constant('paginationLimits', [
+        [10, "Limit 10"]
```

Comment:
I think it's fine if we don't, because it's very likely that people will want more than just one option for limits